content/en/docs/examples/multicluster/gke/index.md

```diff
@@ -237,6 +237,7 @@ $ kubectl label secret ${CLUSTER_NAME} istio/multiCluster=true -n ${NAMESPACE}
     $ kubectl config use-context "gke_${proj}_${zone}_cluster-2"
     $ kubectl apply -f @samples/bookinfo/platform/kube/bookinfo.yaml@ -l service=ratings
     $ kubectl apply -f @samples/bookinfo/platform/kube/bookinfo.yaml@ -l service=reviews
+    $ kubectl apply -f @samples/bookinfo/platform/kube/bookinfo.yaml@ -l account=reviews
```

Comment:
This would require an additional two lines in the bookinfo.yaml:
```
apiVersion: v1
kind: ServiceAccount
metadata:
  name: bookinfo-reviews
  labels:    <===
    account: reviews <===
---
``

Reply:
https://github.com/istio/istio/pull/17577